airbyte-webapp/package.json

```diff
@@ -22,7 +22,7 @@
     "react-checkbox-tree": "^1.6.0",
     "react-code-input": "^3.9.0",
     "react-copy-to-clipboard": "^5.0.2",
-    "react-dom": "npm:@hot-loader/react-dom",
```

Comment:
Actually we do not want to use react-dom. We want to use hot-loader/react-dom from https://www.npmjs.com/package/react-hot-loader